### ISSUE TITLE
The lighting feels off in open gl mode 

### ISSUE THREAD

**Issue by nbk935** [fgsfdsfgs/prboom-plus#6]

**Comment by fgsfdsfgs** [comment]
Yeah well, that's because it IS off. GLBoom doesn't exactly have the best renderer.
For now I can only offer you to try and compare "GLBoom" and "GZDoom" lighting modes to see which one you think is better, you can switch those in the Settings menu in the "OpenGL" section.
You can also play in software to get the visuals 1:1 to how they would be in vanilla.

**Comment by nbk935** [comment]
It's okay i don't mind and in software it runs eh i can do with the lighting on open gl i thought I would point it out  and I love the work you do

**Comment by fgsfdsfgs** [comment]
Software should run at full speed in resolutions up to 480x272, if you go higher it will run like ass.
